recipe/meta.yaml

```diff
@@ -1,14 +1,14 @@
 {% set name = "ply" %}
-{% set version = "3.10" %}
-{% set sha256 = "dd88825a5913714b0c7ec5c7f31b26a5d84862fec0a990c4d9ed88db17937976" %}
+{% set version = "3.8" %}
+{% set sha256 = "e7d1bdff026beb159c9942f7a17e102c375638d9478a7ecd4cc0c76afd8de0b8" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   fn: {{ name }}-{{ version }}.tar.gz
-  url: https://github.com/dabeaz/{{ name }}/archive/{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Comment:
This is not a problem, but there was no reason to change this.

Reply:
The CI got mad about this b/c it couldn't find the appropriate source tarball on github; maybe the `3.8` version is at a different spot?

Reply:
Ahh fair